src/notation/view/widgets/editstyle.cpp

```diff
@@ -2588,6 +2605,13 @@ void EditStyle::textStyleChanged(int row)
     styleName->setEnabled(int(tid) >= int(TextStyleType::USER1));
     resetTextStyleName->setEnabled(styleName->text() != TConv::translatedUserName(tid));
 
+    tupletUseSymbols->setVisible(int(tid) == int(TextStyleType::TUPLET));
+    resetTupletUseSymbols->setVisible(int(tid) == int(TextStyleType::TUPLET));
+    row_textStyleMusicalSymbolsScale->setVisible(int(tid) == int(TextStyleType::TUPLET)
+                                                 || int(tid) == int(TextStyleType::OTTAVA) || int(tid) == int(TextStyleType::PEDAL)
+                                                 || int(tid) == int(TextStyleType::HARP_PEDAL_DIAGRAM));
+    row_textStyleMusicalSymbolsScale->setEnabled(int(tid) != int(TextStyleType::TUPLET) || tupletUseSymbols->isChecked());
```

Comment:
Seems that the `int()` conversions are not necessary here (the reason they are used above is probably the `>=` operator)